openghg_defs/__init__.py

```diff
@@ -6,5 +6,4 @@
 
 site_info_file: Path = data_path / "site_info.json"
 species_info_file: Path = data_path / "species_info.json"
-domain_info_file: Path = data_path / "domain_info.json"
-
+domain_info_file: Path = data_path / "domain_info.json"
```

Comment:
You have deleted the empty line from the end of this file.

Reply:
Fixed.